sys-block/arcconf/arcconf-2.06.23167.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2018 Gentoo Foundation
```

Comment:
why touch that ebuild for Copyright only?

Reply:
That was a misstake. Sorry.